Sources/Hummingbird/Server/Request.swift

```diff
@@ -55,6 +55,34 @@ extension Request {
             throw error
         }
     }
+
+    /// Decode request query using ``URLEncodedFormDecoder``.
+    /// - Parameters
+    ///   - type: Type you want to decode to
+    ///   - context: Request context
+    public func decodeQuery<Type: Decodable>(as type: Type.Type, context: some RequestContext) throws -> Type {
```

Comment:
I think it's a _little_ awkward to discover this API. Can we possibly add it on `URI`?

Reply:
Moved to URI